### DISCUSSION TITLE
How to deploy single application

### DISCUSSION THREAD

**Discussion by richarddavenport** [pnpm/pnpm#5248] [Q&A] [answered]
Pardon my ignorance here... I'm setup using pnpm in a workspace with turborepo. Everything runs great locally, installs, builds, running the apps. My problem is deploying.

A simple version of what I have:
```
.
|-- apps
|   |-- app1
|   |   |-- node_modules
|   |   |   |--node-fetch  <-- has "dependency" on "data-uri-to-buffer"
|   |   `-- src
|-- packages (removed for brevity)
|-- node_modules <-- root node_modules
|   |-- (lots of interesting stuff here)
|-- package.json
|-- pnpm-lock.yaml
|-- pnpm-workspace.yaml
`-- turbo.json
```

The main problem I'm having is that when I deploy, I am only deploying `./apps/app1` so when `node-fetch` runs it can't find the `data-uri-to-buffer` module because it wasn't deployed.

I could really use some help, it seems like I'm missing something really obvious here.

**Comment by zkochan** [comment, marked as answer]
Use the [`pnpm deploy`](https://pnpm.io/cli/deploy) command.

**Reply by richarddavenport** [reply]
Thank @zkochan I had to tweak my `files` in my package.json and it works now. The documentation is a little confusing given that I'm not using docker.

Are you open to PR's to enhance the documentation?

**Reply by zkochan** [reply]
yes
